expressEngine/model/collection/orderItems.js

```diff
@@ -0,0 +1,23 @@
+import mongoose from 'mongoose';
+const { Schema } = mongoose
+
+const orderItem = new Schema({
+    orderId: {
+        type: String,
+        unique:true
+    },
+    productId: {
+        type: String,
+        unique:true
+    },
+    unitPrice: {
+        type: Number,
+    },
+    qunatity: {
+        type: Number
+    }
+})
```

Comment:
why are you not using required field as true in the defined schema? Can you please explain me?